pkg/ccn-lite/ccn-lite-riot.h

```diff
@@ -12,6 +12,7 @@
 /**
  * @defgroup    pkg_ccnlite CCN-Lite stack
  * @ingroup     pkg
+ * @ingroup     net
```

Comment:
From the documentation point of view, multi groups for pkg makes sense to me: are Openthread, lwIP, emb6 network packages ? Yes, so they should appear in Networking root group and package group.
Otherwise, we can create subgroups of pkgs but it will become a mess.